ansible/docker.yml

```
@@ -24,7 +24,7 @@
   tasks:
     - name: get swarm status
       shell: >
-        set -o pipefail && docker info | egrep '^Swarm: ' | cut -d ' ' -f 2
+        set -o pipefail && docker info | grep 'Swarm' | sed 's/\s*Swarm\:\s//'
```

Seems like the sed regex is expecting a `:` after Swarm, so I was thinking the grep could also have this to make it more specific.  Also was wondering if would make sense to do `\s*` after `Swarm\:` in the sed regex.

```suggestion
        set -o pipefail && docker info | grep 'Swarm:' | sed 's/\s*Swarm\:\s*//'
```